cmd/kubeadm/app/images/images.go

```diff
@@ -91,6 +93,7 @@ func GetControlPlaneImages(cfg *kubeadmapi.ClusterConfiguration) []string {
 
 	// start with core kubernetes images
 	if cfg.UseHyperKubeImage {
+		klog.Warningln(`WARNING: DEPRECATED use of the "hyperkube" image for the Kubernetes control plane`)
```

Comment:
can we give more context here ? maybe add the planned removal ? users might expect that we'll maintain this behavior for 3 releases

Reply:
i though that we are going to remove it in v1beta3 right away and if users want to use hyperkube they should stay on v1beta2?

the message can indicate that?